an-apluss/AutoMart
# #166217673 Setup and integrate travisCI, codeclimate and coverall

#### What does this PR do?
- Setup and integrate travisCI, codeclimate and coverall

#### Description of Task to be completed?
- install necessary dependencies for test coverage
- create and setup .travis.yml file
- update .gitignore file with build output
- update README.md file with necessary badges markdown
- delivers[#166217673]

#### How should this be manually tested?
- Clone the repository using `git clone https://github.com/an-apluss/Automart.git` in your terminal
- run `npm install` on your terminal
- run `npm run start` to start up the server
- run `npm test` to see test results

#### What are the relevant pivotal tracker stories?
- PT Story link - ( [166217673](https://www.pivotaltracker.com/story/show/166217673) )

#### Screenshots(if appropriate)
-N/A
